<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="urlEdit">
        <property name="text">
         <string>https://bing.com</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="gotoBtn">
        <property name="text">
         <string>go</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>网页缩放</string>
    </property>
    <addaction name="zoomInAction"/>
    <addaction name="zoomOutAction"/>
    <addaction name="resetZoomAction"/>
   </widget>
   <widget class="QMenu" name="lookUpMenu">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="soucreCodeAction"/>
    <addaction name="setMainPageAction"/>
    <addaction name="historyAction"/>
   </widget>
   <addaction name="lookUpMenu"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="zoomInAction">
   <property name="text">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="resetZoomAction">
   <property name="text">
    <string>重置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="soucreCodeAction">
   <property name="text">
    <string>查看源代码</string>
   </property>
  </action>
  <action name="setMainPageAction">
   <property name="text">
    <string>主页设置</string>
   </property>
  </action>
  <action name="historyAction">
   <property name="text">
    <string>查看历史记录</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
